cmd/pipecd/server.go

```diff
@@ -69,7 +69,8 @@ var (
 )
 
 const (
-	defaultPipedStatHashKey = "HASHKEY:PIPED:STATS"
+	defaultPipedStatHashKey    = "HASHKEY:PIPED:STATS"
+	apiKeyLastUsedCacheHashKey = "HASHKEY:PIPED:API_KEYS" //nolint:gosec
```

Comment:
Just want to confirm, do we still need this `nolint` annotation?

Reply:
Yes, CI golang-cli fails without this nolint